cfpb/consumerfinance.gov
# Paying for College - Financial Path bug fixes

This PR fixes a few minor bugs in Paying for College: Your Financial Path to Graduation. See "Changes" for details.

## Changes
- Graduate students no longer see living situation options (see internal issue 598).. Choosing a living situation is no longer required for graduate students to proceed.
- When a user goes back to choose a different school after proceeding through the app, the app now zeroes the financial costs of the school, and it clears the "costs choice" so the user is again able to select costs from our data.

## How to test this PR
1. Pull it down, get it running, and go to: http://localhost:8000/paying-for-college/your-financial-path-to-graduation/
2. Test to see if the above fixes (under "Changes") are working as you expect!

## Checklist
